packages/sgug-rpm-config/SOURCES/macros.ldconfig

```diff
@@ -0,0 +1,7 @@
+%ldconfig /usr/sgug/bin/echo
+%ldconfig_post(n:) /usr/sgug/bin/echo
+%ldconfig_postun(n:) /usr/sgug/bin/echo
+%ldconfig_scriptlets(n:) %{?ldconfig:\
+%ldconfig_post /usr/sgug/bin/echo \
+%ldconfig_postun /usr/sgug/bin/echo }\
+}
```

Comment:
If the packages are following Fedora guidelines, this version should work perfectly: https://src.fedoraproject.org/rpms/redhat-rpm-config/blob/master/f/macros.ldconfig

Reply:
That'd work great except there's no ldconfig on IRIX.

Reply:
Note that the `%ldconfig` macro is disabled, making everything else empty.